lib/localize/index.js

```diff
@@ -34,7 +34,9 @@ module.exports = function( i18n ) {
 			},
 
 			render: function() {
-				var props = assign( {}, this.props, i18nProps );
+				var props = assign( {
+					locale: i18n.getLocaleSlug()
```

Comment:
Any reason to have `locale` defined in the new object instead of as a prop in `i18nProps` like we do for the rest of the props we pass to the wrapped component?

Reply:
Good question. This was what I actually tried first, but then I realized that `18n.getLocaleSlug()` was only executed once, and the slug would be forever `en` and the shallow comparison wouldn't pick up any changes.

`18n.getLocaleSlug()` is a fairly cheap function to execute (it just returns a value with no computation).

Reply:
Ah, that makes sense 👍 thanks!